### PR TITLE
Changed Ghost Button to work on blue and gray backgrounds

### PR DESCRIPTION
**Pull request by vladmoroz**
New Ghost Button design

### CODE REVIEW

File: packages/radix/src/components/GhostButton.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ export const GhostButton = React.forwardRef<HTMLButtonElement, GhostButtonProps>
               lineHeight: theme.lineHeights[2],
               height: theme.sizes[5],
               minWidth: theme.sizes[5],
+              mixBlendMode: 'multiply',
```
Comment (vladmoroz) [inline packages/radix/src/components/GhostButton.tsx]: I've got the power GIF

File: packages/radix/src/components/GhostButton.tsx [hunk 2]
```diff
@@ -58,19 +59,20 @@ export const GhostButton = React.forwardRef<HTMLButtonElement, GhostButtonProps>
...
-                  backgroundColor: theme.colors.blue300,
-                  boxShadow: `0 0 0 1px ${theme.colors.blue300}`,
+                  backgroundColor: theme.colors.gray300,
+                  boxShadow: `0 0 0 1px ${theme.colors.gray300}`,
+                },
+                focus: {
+                  boxShadow: `0 0 0 2px ${theme.colors.gray500}`,
                 },
-                focus: { boxShadow: `0 0 0 2px ${theme.colors.blue500}` },
               },
```
Comment (vladmoroz) [inline packages/radix/src/components/GhostButton.tsx]: Blue didn't look right for the active state after gray hovers

File: packages/radix/src/components/GhostButton.tsx (hunk 1 above)
Comment (peduarte) [reply packages/radix/src/components/GhostButton.tsx]: lol can we do that in Modulz?

Comment (vladmoroz) [reply packages/radix/src/components/GhostButton.tsx]: I think we should, shouldn't we?

Comment (peduarte) [reply packages/radix/src/components/GhostButton.tsx]: That’s probably the only setting designers will know lol

**Pull request merged**